### DISCUSSION TITLE
Crashing when injecting

### DISCUSSION THREAD

**Discussion by ArexDead** [YimMenu/HorseMenu#66] [General]
Is it normal for RDR2 to crash every time you try to inject a DLL? Used Xenos and little injector with both family menu and the horse menu the result was the same straight crash.

**Comment by maybegreat48** [comment]
I have no idea what your "family menu" is, but I'd recommend not loading it along with HorseMenu

**Reply by ArexDead** [reply]
It's just another dll and I tried to inject it since horsemenu kept crashing rdr2 and I tried to see if horsemenu was the problem or the injector the one I was using was Xenos but it seems it's currently useless since it keeps crashing no matter the DLL file. what other alternative do I have besides Xenos? for the time being I cant inject horsemenu with xenos despite it being on the latest version.

**Comment by Tayipc38** [comment]
Notifications system is suspected to have some issues that might result in crashes upon inject. Currently, I do not have the time to address or confirm this.

**Reply by ArexDead** [reply]
still crashed with notifications off. Used extreme injector v3.

**Comment by MarkEcza** [comment]
@ArexDead Crash on injection ?

**Reply by ArexDead** [reply]
yes crash on injection, currently the native injector of family menu works but if I try another injector the game just crashes and thats it, no matter the DLL or injector.

**Reply by MarkEcza** [reply]
Can you send a log?

**Reply by ArexDead** [reply]
Sure, here you have it
[Xenos.log](https://github.com/YimMenu/HorseMenu/files/13453899/Xenos.log)

**Reply by MarkEcza** [reply]
This is not the HorseMenu log. Its in Documents->Hell Base

**Reply by ArexDead** [reply]
Yeah it doesn't even generate a Hell Base folder it just crashes at that's it no log no anything

**Comment by ArexDead** [comment]
Also if you could tell me which injector do you guys use it would be nice.

**Reply by maybegreat48** [reply]
I use Xenos too, which works well for me. Can you try changing your renderer to Vulkan?

**Reply by ArexDead** [reply]
I tried on both renderers and  got the same result so yeah idk what to do now.

**Comment by ArexDead** [comment]
Well since my problem is only 1 in a billion I decided for the time being only to use family menu and forget about Horse Menu for now hope in a next update is usable for me.
